using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,249 @@
+
+The same argument must also be supplied to the [`add_drop_chunks_policy`
+function][api-add-drop-chunks] when creating a data retention policy for a
+hypertable with a continuous aggregate.  
+
+---
+
+### Best Practices [](best-practices)
+
+**Using `timescaledb.information` Views** 
+The various options used to create the continuous aggregate view, as well as its
+definition can be found in the [`timescaledb_information.continuous_aggregates`
+view][continuous-aggregates-info], and information about the state and progress
+of the materialization background worker jobs can be found in the
+[`timescaledb_information.continuous_aggregate_stats`
+view][continuous-aggregate-stats]. These views can be quite useful for
+administering continuous aggregates and tuning other options noted below. 
+
+**Tuning `refresh_lag` and `refresh_interval`** 
+When designing continuous aggregates, we wanted to avoid write amplification as
+that could drastically affect insert rate for write heavy timeseries workloads.
+We also wanted to deal gracefully with out of order data. There are several
+tunable parameters that allow you to control how quickly data is materialized
+after insert and how far behind the aggregates will remain (so as not to cause
+the same materialization to be modifiied multiple times and cause write amplification). 
+
+We usually stay at least one `bucket_width` (the first argument to the
+`time_bucket` call in the view definition) behind the maximum inserted time
+value. The `refresh_lag` parameter determines how much further behind the
+highest inserted time value the background worker will attempt to materialize
+(you will be behind by `refresh_lag` + `bucket_width`).  So in our example, if
+my `refresh_lag` is 1 hour and the `bucket_width` is 1 hour, the materialization
+will generally be 2 hours behind the maximum inserted value. Tuning the
+`refresh_lag` parameter lower, or even negative, will mean that the aggregates
+will follow inserts more closely, but can cause some write amplification which
+may degrade insert performance. 
+
+The `refresh_interval` controls how frequently materialization jobs will be
+launched by the background worker scheduler. Setting a shorter interval will
+mean materializations happen more "eagerly" but can consume background worker
+resources while it is running.
+
+
+**Modifying the Materialization Hypertable**
+Advanced users may find the need to modify certain properties of the
+materialization hypertable (ie chunk size etc.) or to create further indexes,
+you can find the name of the materialization hypertable in the
+`timescaledb_information.continuous_aggregates` view ([API Docs][continuous-aggregates-info]) like so:
+```sql
+SELECT * FROM timescaledb_information.continuous_aggregates;
+```
+And then modify the materialization hypertable as if it were a normal
+hypertable. For instance, if you have not set the `WITH` option
+[`timescaledb.create_group_indexes][create-continuous-agg] to `false`, then
```

Comment:
this example is weird here because timescaledb.create_group_indexes can only be set when creating the view not modifying. We should use another example here and move this into the create section.